CHANGELOG.md

```diff
@@ -14,3 +14,4 @@ First release! What's possible with this first release:
 - Manipulate data in datasets.
 - Inspect a user's permissions w.r.t. a given Resource. (Experimental.)
 - Retrieve, delete and/or write any file (including non-RDF) from/to a Pod.
+- Fetch information about a Resource (file or dataset) without fetching the resource itself
```

Comment:
I don't mind too much, but given that this is an initial release, I think I'd just lump this under "Manipulate data in datasets" (or maybe make that "Read and manipulate").

Reply:
Resolved in 49a62169510b1fb6c2cacb3a544d10b73d401be3